src/thunderbots/software/grsim_communication/main.cpp

```diff
@@ -29,11 +30,8 @@ void primitiveUpdateCallback(const thunderbots_msgs::PrimitiveArray::ConstPtr& m
 {
     std::vector<std::unique_ptr<Primitive>> primitives;
     thunderbots_msgs::PrimitiveArray prim_array_msg = *msg;
-    for (const thunderbots_msgs::Primitive& prim_msg : prim_array_msg.primitives)
-    {
-        primitives.emplace_back(AI::Primitive::createPrimitiveFromROSMessage(prim_msg));
-    }
-
+    primitives.emplace_back(
+        new PivotPrimitive(0, world.ball().position(), Angle::threeQuarter(), 0.3));
```

Comment:
Hmm, methinks we probably don't want to commit this (hopefully in the future we'll have simulation integration tests that we can use in place of changes like this......)

Reply:
I left it in so that anyone reviewing can run it, will take it out now

Reply:
makes sense, I did indeed use it

Reply:
still here?

Reply:
removed now, forgot to remove in the last one